packages/shared/components/Select/Select.story.tsx

```diff
@@ -0,0 +1,88 @@
+/*
+Copyright 2019 Gravitational, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+import React from 'react';
+import styled from 'styled-components';
+import { Flex, Box, LabelInput } from 'design';
+import Select, { DarkStyledSelect } from '../Select';
+
+export default {
+  title: 'Shared/Select',
+};
+
+export const SelectTypes = () => {
+  return (
+    <Flex justifyContent="space-evenly">
+      <SelectDefault {...props} />
+      <SelectDark {...props} />
+    </Flex>
+  );
+};
+
+const props = {
+  value: [
+    { value: 'admin', label: 'admin' },
+    { value: 'testrole', label: 'testrole' },
+  ],
+  onChange: () => null,
+  options: [
+    { value: 'Relupba', label: 'Relupba' },
+    { value: 'B', label: 'B' },
+    { value: 'Pilhibo', label: 'Pilhibo' },
+  ],
+};
+
+function SelectDefault({ value, onChange, options }) {
+  return (
+    <Box mb="4" width="500px">
+      <LabelInput>User Roles</LabelInput>
+      <Select
+        value={value}
+        onChange={onChange}
+        options={options}
+        isMulti={true}
+      />
+    </Box>
+  );
+}
+
+function SelectDark({ value, onChange, options }) {
+  return (
+    <Box mb="4">
+      <LabelInput>User Roles</LabelInput>
+      <StyledSelect width="500px">
+        <Select
+          value={value}
+          onChange={onChange}
+          options={options}
+          isMulti={true}
+        />
+      </StyledSelect>
+    </Box>
+  );
+}
+
+const StyledSelect = styled(DarkStyledSelect)(
+  ({ theme }) => `
+  .react-select__control,
+  .react-select__control--is-focused {
+    min-height: 40px;
+  }
+  .react-select__input {
+    color: ${theme.colors.text.primary}
+  }
```

Comment:
this should be a dark default, otherwise every time we use `DarkStyledSelect`, we have to repeat this style